lib/api/fields/assets/js/media.js

```diff
@@ -172,15 +172,15 @@
 
 			});
 
-			this.container.on( 'click', '.dashicons-post-trash', function( e ) {
+			this.container.on( 'click', '.dashicon-trash', function( e ) {
```

Comment:
I used the wrong diff to revert this. Fix incoming.

Reply:
@christophherr The original code was using `.dashicons-post-trash`. However, that is the wrong icon, as there is no [post-trash icon](https://www.kevinleary.net/wordpress-dashicons-list-custom-post-type-icons/).  I changed it to [`.dashicons-trash`](https://developer.wordpress.org/resource/dashicons/#trash) in the media scripts, view file, and tests.

Reply:
It's actually [`.dashicons-trash`](https://developer.wordpress.org/resource/dashicons/#trash).  The original code had the wrong icon.